pkg/kubelet/dockertools/manager.go

```diff
@@ -48,12 +48,12 @@ import (
 	"k8s.io/kubernetes/pkg/kubelet/network/hairpin"
 	proberesults "k8s.io/kubernetes/pkg/kubelet/prober/results"
 	"k8s.io/kubernetes/pkg/kubelet/qos"
-	kubetypes "k8s.io/kubernetes/pkg/kubelet/types"
```

Comment:
thats why I did it, I can submit again with it reverted, but about to go to dentist, so just tell me what you guys want.